hermeto/core/package_managers/general.py

```diff
@@ -110,19 +109,7 @@ async def async_download_files(
     :param files_to_download: Dict of files to download with file paths
     :param concurrency_limit: Max number of concurrent tasks (downloads).
     """
-
-    async def on_request_start(
```

Comment:
I think the original intent behind this was "Cachito-Nexus is flakey, so we want to know how many times it tries and fails for each dependency". We probably don't need it. The debug log should still show the failed download

Reply:
So the change alone without a version bump doesn't make the problem go away?